hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: patch
+
+This release significantly improves the data distribution in rule based stateful testing <stateful_testing>,
+by using a technique called Swarm Testing (Groce, Alex, et al. "Swarm testing."
+Proceedings of the 2012 International Symposium on Software Testing and Analysis. ACM, 2012.)
```

Comment:
I'd add a hyperlink to an openly available copy, so people can read it without bouncing off the ACM first - https://www.cs.utah.edu/~regehr/papers/swarm12.pdf I think?